tests/cupyx_tests/scipy_tests/special_tests/test_statistics.py

```diff
@@ -56,7 +59,7 @@ def check_unary_lower_precision(self, name, xp, scp, dtype):
         return self._check_unary(a, name, scp)
 
     @testing.for_dtypes(['e', 'f', 'd'])
-    @testing.numpy_cupy_allclose(atol=atol, rtol=rtol, scipy_name='scp')
+    @testing.numpy_cupy_allclose(atol=1e-6, rtol=1e-6, scipy_name='scp')
```

Comment:
This change unexpectedly relaxes the tolerance for float64. Would you fix the definition of `atol` variable as follows?
```py
atol = {'default': 1e-6, cupy.float64: 1e-14}
```

Reply:
I think we won't need this. I rechecked this, it worked with the default atol tolerance = `1e-14`.

Reply:
Okay!